### PR TITLE
[DOCS] Clarify usage of body overflow hiding re .medium-grid-frame example in docs

### PR DESCRIPTION
**Pull request by paxperscientiam**
## Description
Per PR #11191 and my own [comment](https://github.com/zurb/foundation-sites/issues/11191#issuecomment-389658664), I have added commentary warning re hiding body overflow.

## Motivation and Context
My motivation is to help others that may or may have blindly add `body{overflow:hidden;}` to their code per current suggestion and wind up with a webpage that won't scroll on small screens.

Hope that makes sense!

## Checklist (all required):

- [x] I have read and follow the [CONTRIBUTING](CONTRIBUTING.md) document.
- [x] There are no other pull request similar to this one.
- [x] The pull request title is descriptive.
- [x] The template is fully and correctly filled.
- [x] The pull request targets the right branch (`develop` or `support/*`).
- [x] My commits are correctly titled and contain all relevant information.
- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly to my changes (if relevant).
- [x] I have added tests to cover my changes (if relevant).
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by ncoden — CHANGES_REQUESTED** [review]
TL;DR: XY Grid is already very heavy.

**Comment by ncoden** [comment]
✨ Welcome back @paxperscientiam and thank you for your PR ✨

File: docs/pages/xy-grid.md [hunk 1]
```diff
@@ -304,6 +304,27 @@ To start, add `.grid-frame` to the grid. This sets the grid to be 100vh (the ful
 
 <div class="callout warning">
   Please note to use `.grid-margin-x` or `.grid-margin-y` with `.grid-frame` you need to hide the overflow on the body like so: `body {overflow: hidden;}`.
+  
+  Given the `.medium-grid-frame` example below, you'll want to take care to hide overflow body content for medium and larger screens only.
```
[the remark this answers is not in the document]
Comment (paxperscientiam) [reply docs/pages/xy-grid.md]: Yes. Let me do that.

**Comment by ncoden** [comment]
@paxperscientiam We updated our [CONTRIBUTING](https://github.com/zurb/foundation-sites/blob/develop/CONTRIBUTING.md#contributions) recommendations. Please take a look at the "Contributions" sections, you'll find relevant stuff and advices for this pull request.

**Comment by ncoden** [comment]
Looks good now.

> ed53c10 Update xy-grid.md
> 263bd99 Update xy-grid.md
> 50e1793 Update xy-grid.md
> 680ce8e Update xy-grid.md
> fbf6cfe [docs] Merged two paragraphs.
> d516233 [docs] Removed unncessary code examples

Could you squash commits ?

**Comment by ncoden** [comment]
@paxperscientiam  Looks good now.

> ed53c10 Update xy-grid.md
> 263bd99 Update xy-grid.md
> 50e1793 Update xy-grid.md
> 680ce8e Update xy-grid.md
> fbf6cfe [docs] Merged two paragraphs.
> d516233 [docs] Removed unncessary code examples

Could you squash commits ?

**Comment by paxperscientiam** [comment]
Hey @ncoden , I'm new to squashing, so not sure I did it correctly. Halp?

**Comment by DanielRuf** [comment]
Squashing is basically a `git rebase` with `fixup` for all but the first commit which is reworded with `reword` added to this branch so that there is only one commit.

**Comment by ncoden** [comment]
See https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Review by DanielRuf — APPROVED** [review]

**Comment by ncoden** [comment]
> I just learned from #git on Freenode that this can be done automagically from within GH by whomever does the merging.

True but it can often be useful to get the author commits inside the Foundation branches (instead of squashing all of them), and I'd like to keep the "merge commit + author commit(s)" method consistent.

@DanielRuf I just discoverd that GitHub actually keeps all outdated changes inside their commits, even after the branch is "force pushed", merged and deleted. So we may not have to create new PRs, GitHub keeps everything.

**Comment by DanielRuf** [comment]
I'll rebase your branch later.

Normally you just have to run `git rebase -i HEAD~13` and write `fixup` instead of `pick` for all but not the first commit. And then do `git commit --amend` to reword your first commit or use `reword` instead of `pick`.

**Comment by ncoden** [comment]
Ok, I just seen that this PR was opened from `zurb:master`. I'll clean that.
@paxperscientiam Please create branches on your own fork of `foundation-sites` and open your pull requests from them. Do not make changes directly on the `foundation-sites` repo.

**Comment by ncoden** [comment]
See https://github.com/zurb/foundation-sites/pull/11280

**Comment by paxperscientiam** [comment]
@ncoden My bad and will do!
